core/types/block.go

```diff
@@ -482,6 +489,20 @@ func (b *Block) WithWithdrawals(withdrawals []*Withdrawal) *Block {
 	return block
 }
 
+// WithInclusionList returns a copy of the block containing the given inclusionList.
+func (b *Block) WithInclusionList(addresses []common.Address) *Block {
```

Comment:
If this is a PR to refactor consensus interface, please keep it clean and don't put inclusion lists into this PR

Reply:
done sry